py/selenium/webdriver/chrome/webdriver.py

```diff
@@ -115,6 +115,29 @@ def set_network_conditions(self, **network_conditions):
             'network_conditions': network_conditions
         })
 
+    def execute_cdp_cmd(self, cmd, cmd_args):
+        """
+        Execute Chrome Devtools Protocol command and get returned result
+
+        The command and command args should follow chrome devtools protocol domains/commands, refer to link
+        https://chromium.googlesource.com/chromium/src/+/master/third_party/blink/renderer/devtools/protocol.json
```

Comment:
Might be good to add this link in the docstring as I think it is a better more readable source.
https://chromedevtools.github.io/devtools-protocol/